packages/core/src/utils/spanUtils.ts

```diff
@@ -1,11 +1,23 @@
-import type { Span, TraceContext } from '@sentry/types';
+import type { Span, SpanAttributes, TraceContext } from '@sentry/types';
 import { dropUndefinedKeys, generateSentryTraceHeader } from '@sentry/utils';
 
 /**
  * Convert a span to a trace context, which can be sent as the `trace` context in an event.
  */
 export function spanToTraceContext(span: Span): TraceContext {
-  const { data, description, op, parent_span_id, span_id, status, tags, trace_id, origin } = span.toJSON();
+  const {
```

Comment:
This conflicted with some tests, but I figured it is overall safer as we assumed that this always is a `Span` class instance with `toJSON()` which may be a bit unsafe. Now we just treat this as a POJO which should be safer (and `.toJSON()` will actually go away for v8 as well, so we may as well already do that now)